k8s/ithc/common/ia/case-api.yaml

```diff
@@ -24,6 +24,7 @@ spec:
       ingressHost: ia-case-api-ithc.service.core-compute-ithc.internal
       environment:
         RESTART_ME: "Update due to vault Change"
+        IA_HOME_OFFICE_INTEGRATION_ENABLED: true
```

Comment:
let's put a String as usual:
`IA_HOME_OFFICE_INTEGRATION_ENABLED: "true"`

Reply:
changed to String